src/main/resources/brapi/calls.json

```diff
@@ -14,8 +14,7 @@
           "1.0",
           "1.1",
           "1.2",
-          "1.3",
-          "2.0"
```

Comment:
Why did we remove 2.0 as supported version?

Reply:
Removing 2.0 will resolve the schema mismatch in getCallsResponse.json
![image](https://user-images.githubusercontent.com/55473223/78651191-924a8f00-78f2-11ea-842d-9070e5042e97.png)

Reply:
@nahuel-soldevilla - what do you think about removing 2.0 as supported version in `POST token` call? @gelli-leafnode removed it to resolve schema mismatch error. I guess the BrAPI validator doesn't expect it to be a valid version. Not sure what would be the impact to other consumers if we remove it just to make it pass for the validator.
cc: @mcrimi

Reply:
@darla-leafnode 
I agree to removing it. For v2 we'll need a new endpoint for calls `/v2/serverinfo`

cc @gelli-leafnode @mcrimi